roles/edpm_neutron_metadata/defaults/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+# defaults file for edpm_ovn
+
+# seconds between retries for download tasks
+edpm_neutron_metadata_images_download_delay: 5
+
+# number of retries for download tasks
+edpm_neutron_metadata_images_download_retries: 5
+
+edpm_ovn_neutron_metadata_config_src: /var/lib/openstack/configs/neutron-metadata
+edpm_ovn_neutron_metadata_agent_config_dir: /var/lib/config-data/ansible-generated/neutron-ovn-metadata-agent
+edpm_ovn_metadata_agent_log_dir: "/var/log/neutron"
+
+edpm_ovn_metadata_agent_image: "quay.io/podified-antelope-centos9/openstack-neutron-metadata-agent-ovn:current-podified"
+
+edpm_ovn_metadata_common_volumes:
+  - /lib/modules:/lib/modules:ro
```

Comment:
I'm sure the agent doesn't load kernel modules. (I don't think ovn-controller does either, but that can be addressed separately.)

Reply:
ok, I'll remove it (though that was there before the split also)